tools/mason/MasonInit.chpl

```diff
@@ -227,3 +268,20 @@ proc addSection(sectionName: string, path: string, tomlFile: unmanaged Toml, sho
   generateToml(tomlFile, tomlPath);
   if show then writeln("Added [" + sectionName + "] section to Mason.toml");
 }
+
+proc makeLegalModule(path: string, moduleName: string) throws {
```

Comment:
Perhaps this should just be `createModule`, reiterating the `--moduleName` can be used for other purposes other than using illegal directory names.

Also, can you add a doc-string to this function?